Dockerfile

```diff
@@ -43,21 +43,22 @@ RUN set -eux; \
         alternatives --install "/usr/bin/$base" "$base" "$bin" 20000; \
     done; \
     java -Xshare:dump; \
-    useradd -d /operator -M -s /bin/bash -g root -u 1000 oracle; \
+    groupadd -g 1000 oracle; \
+    useradd -d /operator -M -s /bin/bash -g oracle -u 1000 oracle; \
```

Comment:
wondering if there is any possible side effect for domain on PV where existing PV might have existing files with root group permissions that are no longer accessible for user with 1000 GID.

Reply:
This Dockerfile is for the operator itself and not for WebLogic images. The operator doesn't mount or access PV's for domains.